clang/test/CodeGenSYCL/Inputs/sycl.hpp

```diff
@@ -139,8 +141,9 @@ class accessor {
   _ImplT<dimensions> impl;
 
 private:
-  void __init(__attribute__((opencl_global)) dataT *Ptr, range<dimensions> AccessRange,
-              range<dimensions> MemRange, id<dimensions> Offset) {}
+  void __init(__attribute__((opencl_global_device)) dataT *Ptr,
```

Comment:
What is the value of this change?

Reply:
There is no real value of this change, but to mimic an upcoming LLVM IR change for accessor pointers (aka pointer argument of a generated OpenCL kernel switches address space from 1 to 11).

Reply:
I don't think it is needed. BTW, you will need to update these numbers after https://github.com/intel/llvm/pull/1968 is merged.

Reply:
Removed